assisted-installer.yaml

```diff
@@ -1,24 +1,24 @@
 openshift/assisted-installer:
   revision: c81363a7fada8d23cfbac86ecc0f97835f1fa043
   images:
-  - assisted-installer-controller
-  - assisted-installer
+  - quay.io/ocpmetal/assisted-installer-controller
+  - quay.io/ocpmetal/assisted-installer
 openshift/assisted-service:
   revision: b9ab5d3c77580ab93c29bb43cb78cc6b02d54e88
   images:
-  - assisted-service
+  - quay.io/edge-infrastructure/assisted-service
 openshift-assisted/assisted-ui:
   revision: d60808caecfdb4f6c480dd8e5b07213ea4f210dd
   images:
-  - ocp-metal-ui
+  - quay.io/ocpmetal/ocp-metal-ui
```

Comment:
This is the only one I know hasn't moved.

Reply:
yeah, maybe it also not that interesting
to the best of my knowledge, the UI team has a very different flow for releasing and deploying their image (rather as JS package), plus this repo doesn't use prow

Reply:
okay, it should be handled by UI team very soon. We'll update this location once they finish doing that